File: Algorithms/src/2185. Counting Words With a Given Prefix/Code/solution.java
class Solution {
    public int prefixCount(String[] words, String pref) {
        int count = 0;
        for (String word : words) {
            // Check if the word starts with the prefix
            if (word.startsWith(pref)) {
                count++;
            }
        }
        return count;
    }
}

File: Algorithms/src/2185. Counting Words With a Given Prefix/Code/solution.js
/**
 * @param {string[]} words
 * @param {string} pref
 * @return {number}
 */
var prefixCount = function (words, pref) {
  let count = 0;
  for (let word of words) {
    // Check if the word starts with the prefix
    if (word.startsWith(pref)) {
      count++;
    }
  }
  return count;
};
